src/alto-ui/Popover/Popover.js

```diff
@@ -99,6 +101,7 @@ Popover.propTypes = {
   open: PropTypes.bool,
   includeTarget: PropTypes.bool,
   inert: PropTypes.bool,
+  forwardedRef: PropTypes.object,
 };
 
-export default Popover;
+export default React.forwardRef((props, forwardedRef) => <Popover {...props} ref={forwardedRef} />);
```

Comment:
use import for forwardRef by destucturing `import React, { forwardRef } from 'react';` and use here `export default forwardRef((props, forwardedRef) => <Popover {...props} ref={forwardedRef} />);`

Reply:
Good point, changed